src/pages/Forget-Password/ForgetPassword.jsx

```diff
@@ -10,88 +10,131 @@ import {
     Heading,
     Input,
     Link,
-    Text
+    Text,
+    useToast
 } from "@chakra-ui/react";
 
 function ForgetPassword() {
     const navigate = useNavigate();
+    const toast = useToast();
     const [email, setEmail] = useState("");
-    const { handleToggleColorMode, colors } = customColorMode();
+    const { handleToggleColorMode, colors } = CustomColorMode();
 
-    const handleSubmit = (e) => {
+    const handleSubmit = async (e) => {
         e.preventDefault();
 
+        //checks if email has been filled out
+        if(!email) {
+            toast({
+                title: "Error",
+                description: "Please fill out email",
+                status: "error",
+                duration: 5000,
+                isClosable: true,
+            });
+            return;
+        }
 
+        try {
+            const response = await fetch("/http://localhost:3000/forget-password", { 
```

Comment:
We should move make a service to hold functions for all our API requests. We should also make a file that exports a variable for the base URL so that it's easy to change if we need to.